src/cachemachine/automatedtellermanager.py

```diff
@@ -0,0 +1,47 @@
+"""Manager for all the tellers and their cache deposits."""
+
+__all__ = [
+    "AutomatedTellerManager",
+]
+
+from dataclasses import dataclass, field
+from typing import Dict, List
+
+from aiohttp import web
+from aiojobs import Scheduler, create_scheduler
+from aiojobs._job import Job
+
+from cachemachine.automatedteller import AutomatedTeller
+
+
+@dataclass
+class AutomatedTellerManager:
+    _tellers: Dict[str, AutomatedTeller] = field(default_factory=dict)
+    _jobs: Dict[str, Job] = field(default_factory=dict)
+    _scheduler: Scheduler = None
+
+    async def init(self, app: web.Application) -> None:
+        self._scheduler = await create_scheduler()
```

Comment:
The reason why I did it this way is that I'm using the on_startup and on_cleanup handlers, and reusing the same instance.  This is only ever created in create_app in the app.py.  Since there's also a cleanup which closes the scheduler, I wanted to keep this even, so that the app can call on_startup again and on_cleanup again, although I guess I only need the on_cleanup.  I can fold the scheduler part into the __init__.

Reply:
Yeah, that makes sense and that seems fine too, since it might be confusing for it to have only an `on_cleanup` handler and not an `on_startup` handler.